submission/distributor.go

```diff
@@ -177,6 +177,10 @@ func incRspsCounter(logURL string, endpoint string, rspErr error) {
 		status = http.StatusBadRequest // default to this if status code unavailable
 		if err, ok := rspErr.(client.RspError); ok {
 			status = err.StatusCode
+			if err.Err != nil && status == http.StatusOK {
+				// got SCT response, but it's either cannot be decoded or cannot be verified
+				status = http.StatusVariantAlsoNegotiates
```

Comment:
I have little idea whether `StatusVariantAlsoNegotiates` is appropriate, but regardless of that:

What problem does this solve? The error code is modified only for metrics, which means the metric is hiding the real status code returned by the log server. Whoever looks at this metric needs to remember that "506" actually means "200" + SCT verification fail.

Suggested solutions:
- Since `RspError` is effectively a pair of status code and an arbitrary error, how about having 2 metrics: one for the status code, and one for the arbitrary errors. You could then surface this error as something like `rspsErrorsCounter.Inc(logURL, endpoint, "SCT_VERIF_FAILED")`.
- Use a non-HTTP-code label in the first metric, so that there is no confusion. For example, `rspsCounter.Inc(logURL, endpoint, "SCT_VERIF_FAILED")`.

Reply:
Should I resolve this comment or you prefer to keep it open?